article/app/views/liveblog/liveBlogBodyContent.scala.html

```diff
@@ -30,11 +30,20 @@
             data-test-id="live-blog-blocks"
         >
 
+        @if(model.currentPage.currentPage.blocks.isEmpty) {
+            <div class="live-blog__no-blocks-container">
+                <span class="live-blog__no-blocks-text">
+                    There have not been any key events yet. Please wait for updates.
+                </span>
+            </div>
+        } else {
             @views.html.liveblog.liveBlogBlocks(model.currentPage.currentPage.blocks, article, timezone)
-
+        }
         </div>
 
-        @fragments.liveBlogNavigation(article.content.metadata.id, model.currentPage)
+        @if(model.currentPage.currentPage.blocks.nonEmpty) {
+            @fragments.liveBlogNavigation(article.content.metadata.id, model.currentPage)
```

Comment:
we can move this in the else block above

Reply:
ah ofc

Reply:
fixed